inst/private/doctest_run_tests.m

```diff
@@ -40,25 +40,53 @@
   doctest_datastore('set_current_index', DOCTEST__i);
   DOCTEST__current_test = doctest_datastore('get_current_test');
 
+  DOCTEST__current_test.directive_issue = false;
+
   % define test-global constants (these are accessible by the tests)
   DOCTEST_OCTAVE = is_octave();
   DOCTEST_MATLAB = ~DOCTEST_OCTAVE;
 
   % determine whether test should be skipped
   % (careful about Octave bug #46397 to not change the current value of “ans”)
-  eval (strcat ('DOCTEST__current_test.skip = ', ...
-                 doctest_join_conditions(DOCTEST__current_test.skip), ...
-                ';'));
+  try
+    eval (strcat ('DOCTEST__current_test.skip = ', ...
+                  doctest_join_conditions(DOCTEST__current_test.skip), ...
+                  ';'));
+  catch DOCTEST__exception
+    DOCTEST__current_test.directive_issue = true;
+    DOCTEST__current_test.skip = [];
```

Comment:
`[]` should probably be `false`

Reply:
Or maybe `true`: the test is not run: as far as future tests are concerned it was skipped...